docs/cli/manage.md

```diff
@@ -0,0 +1,5 @@
+# More Coming Soon!
```

Comment:
>+# More Coming Soon!

What is this page meant to be for?

Reply:
I was going pull in the CLI README but wasn't sure if you preferred to not add that yet. We can remove this section for now and decide later.